geomesa-web/geomesa-web-core/src/main/scala/org/locationtech/geomesa/web/core/GeoMesaBaseDataStoreServlet.scala

```diff
@@ -0,0 +1,43 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.web.core
+
+import org.locationtech.geomesa.utils.cache.FilePersistence
+
+trait GeoMesaBaseDataStoreServlet extends GeoMesaScalatraServlet {
```

Comment:
maybe rename this class to be more descriptive - like PersistentDataStoreServlet